### PR TITLE
[aptogotchi] support aip-62 in wallet adapter

### PR DESCRIPTION
**Pull request by angieyth**
change to new wallet adapter implementation after AIP-62

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -11,16 +11,18 @@
...
   },
   "dependencies": {
-    "@aptos-labs/ts-sdk": "^1.5.1",
-    "@aptos-labs/wallet-adapter-react": "^2.1.7",
+    "@aptos-labs/ts-sdk": "^1.13.2",
+    "@aptos-labs/wallet-adapter-react": "^3.0.2",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@radix-ui/react-dialog": "^1.0.5",
+    "@wallet-standard/core": "^1.0.3",
```
Comment (0xmaayan) [inline frontend/package.json:19]: dont think you need it

File: frontend/package.json [hunk 2]
```diff
@@ -11,16 +11,18 @@
...
+    "aptos": ">=1.21.0 <2.0.0",
```
Comment (0xmaayan) [inline frontend/package.json:20]: are you still using old sdk?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@aptos-labs/wallet-adapter-react": "^2.3.1"
```
Comment (0xmaayan) [inline package.json:3]: probably use `3.0.2`

File: frontend/package.json [hunk 4]
```diff
@@ -11,16 +11,18 @@
     "gen-abi": "./gen_abi.sh"
   },
   "dependencies": {
-    "@aptos-labs/ts-sdk": "^1.5.1",
-    "@aptos-labs/wallet-adapter-react": "^2.1.7",
+    "@aptos-labs/ts-sdk": "^1.13.2",
```
Comment (0xmaayan) [inline frontend/package.json:14]: can we use latest? 1.17.0?

**Review by Mcilz — APPROVED** [review]

**Pull request closed without merging**